src/scicookie/profiles/base.yaml

```diff
@@ -61,12 +61,12 @@ project_license:
   default: BSD 3 Clause
   # first choice is the default for the UI
   choices:
-    - BSD 3 Clause
-    - MIT
-    - ISC license
-    - Apache Software License 2.0
-    - GNU General Public License v3
-    - Other
+  - BSD 3 Clause
+  - MIT
+  - ISC license
+  - Apache Software License 2.0
+  - GNU General Public License v3
+  - Other
```

Comment:
This should not have been a part of this PR.

Reply:
I removed blue from here, so it is showing rest as added, I guess

Reply:
Nope, the indentation is wrong. Maybe your IDE formatted them with the rules you defined.